src/ixdat/config.py

```diff
@@ -28,9 +41,62 @@ def ixdat_temp_dir(self):
         return temp_dir
 
 
-CFG = Config()
+CFG = _Config()
 
 
 def prompt_for_permission(prompt):
     yn = input(prompt + "\nEnter y for yes or anything else for no.")
     return yn in ["y", "yes", "Y", "Yes"]
+
+
+class _PluginOptions:
```

Comment:
I small concern is whether we should standardize on whether these global objects are capital or not.

Reply:
I can see that you already settled on something whether the actual names of the settings are capital

Reply:
I was thinking to use the standard PEP-8 convention of CamelCase for class and snake_case for object. Maybe `CFG` should change to `config` (`from ixdat.config import config`)? But I think it's also fine to leave it.

Reply:
Yeah. I would like to keep it pep8-ish as well. It was just because you upper cased the properties on the class, I thought maybe it was on purpose. But I would be a favor both of changing that, so it would be:
```python
plugins.activate_quant = True
```

and `CFG` renamed to `config`.

Reply:
Done, with a deprecation decorator around `plugins.USE_QUANT` to be nice to Bethan and Daisy.

Reply:
Small change to your suggestion:
I've actually renamed the property `use_quant`, removed its setter, and added methods `activate_quant()` and `deactivate_quant()` because it seems more grammatically correct.